Misc/NEWS.d/next/Core and Builtins/2023-02-10-07-21-47.gh-issue-101765.MO5LlC.rst

```diff
@@ -0,0 +1 @@
+Fix SystemError / segmentation fault in iter ``__reduce__`` when calling ``__builtins__.__dict__["iter"]`` mutates the iter object.
```

Comment:
nit: "calling" is an unusual term for subscripting a dictionary; I think it could just be eliminated here.
```suggestion
Fix SystemError / segmentation fault in iter ``__reduce__`` when ``__builtins__.__dict__["iter"]`` mutates the iter object.
```

Reply:
Yeah I agree, also I think I'll remove the `iter` key specifically since this now affects `reversed` as well. How about

> Fix SystemError / segmentation fault in iter ``__reduce__`` when internal access of ``builtins.__dict__`` keys mutates the iter object.